File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/todo/dto/GetTodosResponse.java
package com.rolandvitezhu.todocloud.network.api.todo.dto;

import com.google.gson.annotations.SerializedName;
import com.rolandvitezhu.todocloud.data.Todo;
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse;

import java.util.ArrayList;

public class GetTodosResponse extends BaseResponse {

  @SerializedName("todos")
  public ArrayList<Todo> todos;

  public ArrayList<Todo> getTodos() {
    return todos;
  }

  public void setTodos(ArrayList<Todo> todos) {
    this.todos = todos;
  }
}

File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/todo/dto/GetTodosResponse.kt
package com.rolandvitezhu.todocloud.network.api.todo.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.rolandvitezhu.todocloud.data.Todo
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse
import kotlinx.android.parcel.Parcelize
import java.util.*

@Parcelize
data class GetTodosResponse (
        @SerializedName("todos")
        var todos: ArrayList<Todo?>?
) : Parcelable, BaseResponse() {
    constructor() : this(
            null
    )
}